test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -578,7 +578,7 @@ Describe "Help failure cases" -Tags Feature {
 
         # under some conditions this does not throw, so include what we actually got
         $helpTopic = [guid]::NewGuid().ToString("N")
-        { & $command $helpTopic -ErrorAction Stop } | Should -Throw -ErrorId "HelpNotFound,Microsoft.PowerShell.Commands.GetHelpCommand" -Because "$(& $command $helpTopic -ea Ignore)"
+        { & $command $helpTopic -ErrorAction Stop } | Should -Throw -ErrorId "HelpNotFound,Microsoft.PowerShell.Commands.GetHelpCommand" -Because "A help topic was found for $helpTopic"
```

Comment:
```suggestion
        { & $command $helpTopic -ErrorAction Stop } | Should -Throw -ErrorId "HelpNotFound,Microsoft.PowerShell.Commands.GetHelpCommand" -Because "A help topic was not found for $helpTopic"
```

Reply:
@PaulHigin  - actually in this case, it's an error if we _find_ a help topic

Reply:
Yes, but the test expectation is a thrown error because the topic is not found, and I feel the 'Because' text should make that clear.